deploy/helm/values.yaml

```diff
@@ -453,6 +453,15 @@ trivy:
   #   - CVE-1970-0001
   #   - CVE-1970-0002
 
+  # -- configFile can be used to tell Trivy to use specific options available only in the config file (ex. Mirror registries).
+  configFile: ~
```

Comment:
does the `~` imply anything?

Reply:
The tilde is one of the ways the null value can be written. 
I thought it's a common style for this value.yaml:
https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L187-L188